components/brave_rewards/resources/ui/components/walletWrapper/index.tsx

```diff
@@ -702,9 +705,13 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
                       </StyledGrantWrapper>
                       : null
                   }
-                  <StyledActionWrapper>
-                    {this.generateActions(actions, id)}
-                  </StyledActionWrapper>
+                  {
+                    !onlyAnonWallet
```

Comment:
we need to keep this part as is, because now we don't have settings link in the panel anymore

Reply:
done